src/Go/boardState/goStyles.ts

```diff
@@ -29,31 +37,56 @@ export const pointStyle = makeStyles<unknown, Size | Point | Structure | Highlig
     hover: {},
     valid: {},
     priorPoint: {},
+    cyberStyle: {},
+    highlightText: {},
+    hack: {},
+    hp: {},
+    money: {},
+    int: {},
+    cha: {},
     point: {
       position: "relative",
       height: "100%",
       width: "100%",
 
-      [`&.${classes.hover}.${classes.valid}:hover .${classes.innerPoint}`]: {
+      [`&.${classes.hover}.${classes.valid}.${classes.cyberStyle}:hover .${classes.innerPoint}`]: {
         outlineColor: theme.colors.white,
       },
-      [`&.${classes.hover}.${classes.priorPoint} .${classes.innerPoint}`]: {
+      [`&.${classes.hover}.${classes.cyberStyle}.${classes.priorPoint} .${classes.innerPoint}`]: {
         outlineColor: theme.colors.white,
       },
-      [`&.${classes.hover}.${classes.priorPoint} .${classes.priorStoneTrad}.${classes.blackPoint}`]: {
-        outlineColor: theme.colors.white,
-        display: "block",
-      },
-      [`&.${classes.hover}.${classes.priorPoint} .${classes.priorStoneTrad}.${classes.whitePoint}`]: {
-        outlineColor: theme.colors.black,
-        display: "block",
-      },
+      [`&.${classes.hover}.${classes.cyberStyle}.${classes.priorPoint} .${classes.priorStoneTrad}.${classes.blackPoint}`]:
+        {
+          outlineColor: theme.colors.white,
+          display: "block",
+        },
+      [`&.${classes.hover}.${classes.cyberStyle}.${classes.priorPoint} .${classes.priorStoneTrad}.${classes.whitePoint}`]:
+        {
+          outlineColor: theme.colors.black,
+          display: "block",
+        },
       [`&.${classes.hover}:hover .${classes.coordinates}`]: {
         display: "block",
       },
       [`&:hover .${classes.broken}`]: {
         opacity: "0.4",
       },
+
+      [`&.${classes.hack} .${classes.innerPoint}`]: {
```

Comment:
What is this for?

Edit: I see what it is for. IMO, this is needless complexity (RGB is likely all that is wanted/needed), but if you have any evidence/knowledge to the contrary, I'm OK leaving it in.

Reply:
I wanted to add this so that players can 1) easily match their theme, or 2) slap in some testing values right away without looking up RGB colors. Originally I was going to do only this, since theme-coloring is so much a part of the game's look and feel, but then I added the rgb support later for flexibility.